unit_tests/test_redis_aggregator.py

```diff
@@ -62,44 +96,98 @@ def fixture_mock_client_command_buffer(mocker):
 
 @pytest.fixture(name="aggregator")
 def fixture_aggregator(mocker):
-    mocker.patch("gsy_e_sdk.redis_aggregator.RedisAggregator._create_aggregator")
+    mocker.patch("gsy_e_sdk.redis_aggregator.uuid.uuid4",
+                 return_value=TEST_TRANSACTION_ID)
+    mocker.patch("gsy_e_sdk.redis_aggregator.wait_until_timeout_blocking")
     return RedisAggregator(aggregator_name=TEST_AGGREGATOR_NAME)
 
 
 class TestRedisAggregatorConstructor:
-    """Includes tests for the constructor of the RedisAggregator class."""
+    """Include tests for the constructor of the RedisAggregator class."""
 
     @staticmethod
-    @pytest.mark.usefixtures("mock_grid_fee_calculation")
-    def test_constructor_grid_fee_calculation_instantiated(aggregator):
-        assert aggregator.grid_fee_calculation is not None
+    @pytest.mark.usefixtures("mock_transaction_id_and_timeout_blocking")
+    def test_constructor_grid_fee_calculation_instantiated():
+        grid_fee_calculation_mock = MagicMock()
+        with patch("gsy_e_sdk.redis_aggregator.GridFeeCalculation",
+                   return_value=grid_fee_calculation_mock) as mocked_class:
+            agg = RedisAggregator(aggregator_name=TEST_AGGREGATOR_NAME)
+            mocked_class.assert_called()
```

Comment:
Nit: I have the feeling that this is redundant since the assertion below implies this already.

Reply:
You are right!